<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>afGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
	font-weight: bold;
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="jobsTab">
       <attribute name="title">
        <string>Jobs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="refreshJobTreeButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <item>
           <widget class="QTreeWidget" name="jobTree">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="indentation">
             <number>12</number>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>User</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Progress</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Capacity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Start</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>End</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Length</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time Created</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time Started</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time Done</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Test job</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Test Block</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
             </item>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="taskList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>bf</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Details">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Job Details</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="6" column="1">
              <widget class="QLabel" name="jobMaximumRunningValue">
               <property name="text">
                <string>Maximum Running Tasks Value</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Maximum Running Tasks Per Host</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="jobErrorsValue">
               <property name="text">
                <string>Errors Values</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="jobDependMaskValue">
               <property name="text">
                <string>Depend Mask Value</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Blocks/Tasks</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Maximum Running Tasks</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="jobStatusValue">
               <property name="text">
                <string>Status Value</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Status</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Errors</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="jobMaximumRunningPerHostValue">
               <property name="text">
                <string>Maximum Running Tasks Per Host Value</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Depend Mask</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="jobTasksValue">
               <property name="text">
                <string>Blocks/Tasks Value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="jobNameValue">
               <property name="text">
                <string>Name Value</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Block Details</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QProgressBar" name="blockProgressValue">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="blockNameValue">
               <property name="text">
                <string>Block Name Value</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Status</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Errors</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="blockStatusValue">
               <property name="text">
                <string>Block Status Value</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="blockDependMaskValue">
               <property name="text">
                <string>Block Depend Mask Value</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Depend Mask</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Tasks</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="blockTasksValue">
               <property name="text">
                <string>Block Tasks Value</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="blockErrorsValue">
               <property name="text">
                <string>Block Errors Value</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rendersTab">
       <attribute name="title">
        <string>Renders</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeWidget" name="rendersTree">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Capacity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CPU</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Memory</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Swap</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Network</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HDD</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>OS</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Priority</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>render001</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>40</string>
           </property>
           <property name="text">
            <string>128GB</string>
           </property>
           <property name="text">
            <string>64GB</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Render Details</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Name Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1421</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
